_episodes/05-rdataframe.md

```diff
@@ -33,15 +33,18 @@ ROOT tries to make parallelization as simple as possible for you. For this reaso
 ```python
 import ROOT
 
-# Enable multi-threading
-ROOT.EnableImplicitMT(0)
```

Comment:
That was on purpose so they can later rerun on 2 or more threads and see that they run faster!

Reply:
At the end, all code snipplet should be copy pastable one after another and result in a working program

Reply:
I don't think the idea worked, `EnableImplicitMT(0)` is equivalent to `EnableImplicitMT()` :D

but to keep things copy-pastable the second `EnableImplicitMT` command can be commented out

Reply:
Ah damn, I thought the `0` disables it! So well, now it's a win win situation ;) I'll make a issue and fix it later!